thunder/core/transforms.py

```diff
@@ -1435,6 +1436,32 @@ def _copy_with_setitem_grad(a: TensorProxy, index, value: Number | TensorProxy):
 
 register_grad(pids.COPY_WITH_SETITEM, _copy_with_setitem_grad)
 
+
+# def _log_sigmoid_grad(
```

Comment:
For the follow-up, there's also the question of if we should have a custom grad for log_sigmoid generally, separate from the PyTorch executor's special grad function